modules/vector-index/pages/vector-index-best-practices.adoc

```diff
@@ -0,0 +1,262 @@
+These changes do not affect the QPS or latency of queries that use the index.
+
+*Recommendation:* if the length of time it takes to build the index or the amount of memory the index uses is a concern, you can try increasing the number of partitions.
+
+=== Replicas
+The number of replicas you set affects the fault tolerance, query throughput, and memory footprint of the index.
+You set this value using the `num_replicas` key in the `WITH` clause when you create or alter your index.
+
+
++
+Tests show that increasing the number of replicas learly increases the QPS and linerly reduces the latency of queries that use the index.
+However, it also linearly increased the memory footprint of the index.
+Adding replicas does not affect the time it takes to build the index.
+
+
+*Recommendation:* if you find that your query throughput or latency is not meeting your needs, you can try increasing the number of replicas for the index if you can afford the additional memory usage.
+
+[#quantization]
+=== Quantization
+Vector indexes always use quantization to compress the vectors they index. 
+Couchbase {product-name} supports two quantization methods: xref:vector-index:vectors-and-indexes-overview.adoc#sq[scalar quantization (SQ)] and xref:vector-index:vectors-and-indexes-overview.adoc#pq[product quantization (PQ)].
+You set the quantization method using the `description` parameter in the `WITH` clause when you create your index.
+
+
+Couchbase tested different quantization settings using datasets that differed in size and number of dimensions.  
+The results of these tests appear in the following table:
+
+
+[%autowidth, options="header"]
+|===
+| Dataset | Index Type | Quantization Setting | Build Time | Memory Use | Recall Accuracy | QPS | Latency
+
+| 100 million 128 dimensions
+| Hyperscale
+| SQ8
+| Lowest
+| Highest 
+| Best 
+| Best
+| Best 
+
+| 5 million, 1536 dimensions
+| Composite
+| PQ32x8
+| Slightly higher than SQ8
+| 75% less than SQ8 
+| lower than SQ8 
+| Lower than SQ8
+| Lower than SQ8
+
+|===
+
+
+NOTE: Couchbase also tested Hyperscale Vector indexes containing 1 billion vectors with 128 dimensions using SQ4 quantization. 
+The results showed acceptable recall accuracy and similar performance to SQ8 on smaller datasets.
+
+ 
+*Recommendations:* 
+
+* SQ8 provides the best balance between memory use and recall for lower dimensional datasets. 
+* If you have a low-dimensional dataset that's in the range of a billion vectors, consider using SQ4 quantization.
+* Use PQ for higher-dimensional datasets if you want to reduce memory use and are willing to accept less accurate recall, decreased QPS, and increased latency.
+
+
+// Similarity Function::
+// Sets the function the index uses when determining the similarity of vectors when identifying centroids.
+// Couchbase {product-name} supports two similarity 
+// 
+// Holding off on simlarity because the best practice doc is mute on it, also lacks testing for dot?
+
+
+== Tune Queries
```

Comment:
Also cover topNScan for Hyperscale Vector Index - https://docs.google.com/document/d/10ruCMnZB6pZtqcN_uDn9a9BfLmaaHA_PG5BcLBpFFEU/edit?tab=t.0

Reply:
Note that topNScan is covered in the Vector Indexes reference page: see #407